tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -104,3 +105,32 @@ Is RHODS Version Greater Or Equal Than
     ${comparison} =  GTE  ${ver}  ${target}
     [Return]  ${comparison}
 
+Move To Installed Operator instance
+    [Arguments]   ${operator_name}     ${tab_name}      ${namespace}
+    Switch To Administrator Perspective
+    Navigate to Installed Operators
+    Installed Operators Should Be Open
+    Log To Console   ${namespace}
+    Run Keyword If  "${namespace}" == "None"   Select All Projects
+    ...         ELSE   Select Project By Name   ${namespace}
+    #Select All Projects
+    Click On Searched Operator   ${operator_name}
+    Switch To New Tab       ${tab_name}
+    sleep    5
+
+Create Installed Operator instance
+    [Arguments]    ${operator_name}     ${tab_name}     ${namespace}=None
+    Move To Installed Operator instance    ${operator_name}     ${tab_name}    ${namespace}
+    Click Button     Create ${tab_name}
+    Wait Until Element is Visible     //button[contains(text(), "Create")]          timeout=10
+    Click Button      Create
```

Comment:
Probably we may need a keyword to check the instance status like done in "Wait Until Installation Completes"

Reply:
what i have seen is it doesn't take much time to get to status "Conditions: Initialized, Deployed". If it fails here we will get to know in the next step while launching the notebook. If you feel i can write a code for it

Reply:
mmmh not sure honestly, let's see how it behaves with different operators in future

Reply:
hmm..i have seen same atleast in openvino and aikit